docs/source/install/deb.rst

```diff
@@ -59,6 +59,7 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
 
     sudo apt-get install -y mongodb-org
     sudo apt-get install -y rabbitmq-server
+    sudo apt-get install -y postgresql
```

Comment:
:+1: and Ouch

Luckily, mistake appeared not so long ago, with MongoDB 3.2 changes:
[#213 Update documentation for MongoDB 3.2 changes, reduce duplication / copy & paste](https://github.com/StackStorm/st2docs/pull/213)